src/runtime.js

```diff
@@ -530,7 +530,7 @@ export class MultidocManager {
      * Sets the document's visibility state.
      * @param {!VisibilityState} state
      */
-    amp.setVisibilityState = function(state) {
+    amp['setVisibilityState'] = function(state) {
```

Comment:
@choumx can these be extern'd instead? i'm not sure what shadowRoot.AMP is. is it the same as the top layer AMP?

Reply:
@choumx interface wise though is the top level `AMP` global the same thing as the shadowRoot.AMP ?

Reply:
No, `AMP` and `shadow` in the example above are different objects.

Reply:
@choumx got it. thanks